Projects/Embark/Sources/Viewables/Actions/TextActionSet.swift

```diff
@@ -35,18 +46,28 @@ extension TextActionSet: Viewable {
                 shouldAutoFocus: index == 0,
                 fieldStyle: .embarkInputSmall
             )
-            return (signal: view.addArranged(input), shouldReturn: input.shouldReturn, action: textAction)
+            
+            let isFirstName: Bool = index == 0
```

Comment:
Isn't what you should do here is instead `isLastTextAction` and !isLastTextAction render a divider? :) I think text action can have any number of inputs, so there might be cases where we have 3 instead of two items.